client/pom.xml

```diff
@@ -61,7 +61,7 @@
       <dependency>
         <groupId>com.google.guava</groupId>
         <artifactId>guava</artifactId>
-        <version>15.0</version>
+        <version>23.6-jre</version>
```

Comment:
Let's upgrade guava also in `aet-features.xml` file.

Reply:
/osgi-dependencies/aet-features.xml is recently added to gitignore. Is this expected? That's why my change is not in this PR.